src/freenet/clients/http/staticfiles/themes/winterfacey/theme.css

```diff
@@ -316,48 +330,57 @@ ul.config {
   margin-left: 4ex;
 }
 
-/*
- *
- * Second level
- *
- */
+
+/*** Second level ***/
 #navbar #navlist .subnavlist,
 #navbar #navlist .subnavlist-selected,
 #navbar #navlist .subnavlist-not-selected {
-  background-color: #FFFFFF;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  border-radius: 6px;
-  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+  background-color: #fff;
+  border: 1px solid rgba(0, 0, 0, .2);
+  -webkit-border-radius: 6px;
+    -moz-border-radius: 6px;
+      border-radius: 6px;
+  -webkit-box-shadow: 0 5px 10px rgba(0, 0, 0, .2);
+    -moz-box-shadow: 0 5px 10px rgba(0, 0, 0, .2);
+      box-shadow: 0 5px 10px rgba(0, 0, 0, .2);
   vertical-align: middle;
 }
 
 #navbar #navlist li ul,
 #navbar #navlist li ul li,
 #navbar #navlist li ul li a {
   font-size: 14px;
-  text-transform: none;
   color: #333;
+  text-transform: none;
 }
 
 #navbar ul#navlist li ul li {
   text-align: left;
   border-bottom: 1px solid;
-  border-image: linear-gradient(to right, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 50%, rgba(51,51,51,0) 100%);
+  -o-border-image: -o-linear-gradient(left, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 50%, rgba(51,51,51,0) 100%);
+    -webkit-border-image: -webkit-gradient(linear, left top, right top, from(rgba(0,0,0,0)), color-stop(50%, rgba(0,0,0,1)), to(rgba(51,51,51,0)));
+      border-image: -webkit-gradient(linear, left top, right top, from(rgba(0,0,0,0)), color-stop(50%, rgba(0,0,0,1)), to(rgba(51,51,51,0)));
```

Comment:
the general border-image uses -webkit-gradient here — shouldn’t this be the standard linear-gradient?

Reply:
A few lines are written below the standard `linear-gradient` so it will be overridden. `-webkit` for cross-browser compatibility in case something doesn't work out somewhere.

Reply:
sounds good.